autoPyTorch/pipeline/components/training/trainer/__init__.py

```diff
@@ -66,6 +66,7 @@ def __init__(self,
                          random_state=random_state)
         self.run_summary: Optional[RunSummary] = None
         self.writer: Optional[SummaryWriter] = None
+        self.early_stopping_dataset: Optional[str] = None
```

Comment:
`split_for_earlystop_criteria`?
At least, `dataset` is confusing, so I would recommend to rename it.

Reply:
Sure